.github/workflows/towerfulltest_illumina.yml

```diff
@@ -0,0 +1,27 @@
+name: Tower AWS full size tests Illumina
+# This workflow is triggered on published releases.
+# It can be additionally triggered manually with GitHub actions workflow dispatch button.
+# It runs the -profile 'test_full_hifi' on AWS batch via Tower
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+jobs:
+  run-tower:
+    name: Run AWS full tests Illumina
+    if: github.repository == 'Arcadia-Science/seqqc'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Launch workflow via tower
+        uses: nf-core/tower-action@v3
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
+          compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
+          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/seqqc/work-${{ github.sha }}
+          parameters: |
+            {
+              "outdir" : "s3://${{ secrets.AWS_S3_BUCKET }}/fulltests/results-${{ github.sha }}/illumina"
+            }
+          profiles: test_full_illumina,aws_tower
```

Comment:
what does the aws_tower profile do?

Reply:
docs: https://github.com/nf-core/configs/blob/master/docs/aws_tower.md
config: https://github.com/nf-core/configs/blob/master/conf/aws_tower.config